src/main/java/it/gov/pagopa/payment/service/payment/qrcode/expired/BaseCommonCodeExpiration.java

```diff
@@ -6,14 +6,15 @@
 import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
-public abstract class BaseQRCodeExpiration {
+public abstract class BaseCommonCodeExpiration {
 
     protected final AuditUtilities auditUtilities;
 
-    protected static final String EXPIRED_QR_CODE = "EXPIRED_QR_CODE";
+    private final String expiredCode;
 
-    protected BaseQRCodeExpiration(AuditUtilities auditUtilities) {
+    protected BaseCommonCodeExpiration(AuditUtilities auditUtilities, String expiredCode) {
```

Comment:
If the expiration amount change based on the channel, the query used to extract them should accept also the channel